pages/_app.tsx

```diff
@@ -109,9 +114,16 @@ CustomApp.getInitialProps = async (
     ? 'development'
     : 'production';
 
+  const workflowsPilotUrl = process.env.WORKFLOWS_PILOT_URL;
+
   return {
     ...appProps,
-    pageProps: { ...appProps.pageProps, user, environmentName },
+    pageProps: {
+      ...appProps.pageProps,
+      user,
+      environmentName,
+      workflowsPilotUrl,
```

Comment:
Instead of passing the individual values here, I'd probably create the `appConfig` object in advanced, and pass that through instead. Something like:

```ts
const appConfig = {
  workflowsPilotUrl: process.env.WORKFLOWS_PILOT_URL
};

return {
  ...appProps,
  pageProps: {
    // ...
    appConfig
  }
};
```